frequency-chain/style-guide
# Add Cards component to style guide

# Goal
The purpose of this PR is to add a Card component according to the UX designs.

The Card takes as props 
* an icon Svelte component which assumes it's an SVG,
* a background color
* title
* title color
* content (as slot)
* content color
* class (used as extras)

When I changed the bgcolor it seemed odd not to let you change the text color.  Since the title text color was different from the card text color, it also seemed you should be able to specify that color as well.  The defaults match what is in the design.

Closes #9 
- [x] Add Card component with props
- [x] Add a Story that illustrates how to use the component

## Steps to Verify:
1. `npm run storybook`
2. navigate to Features-->Card and verify that the view looks like the screenshot and also matches up with the Figma designs from the Issue

## Screenshots (optional):
<img width="1165" alt="Screenshot 2024-10-25 at 8 56 04 AM" src="https://github.com/user-attachments/assets/ff02a8cb-dabd-4863-823f-ad8c21b7979a">
